src/platforms/common/configuration/sampling.mdx

```diff
@@ -39,7 +39,7 @@ Changing the error sample rate requires re-deployment. In addition, setting an S
 
 </Note>
 
-<PlatformSection notSupported={["elixir", "javascript.electron", "perl", "rust", "native"]}>
+<PlatformSection notSupported={["elixir", "javascript.electron", "perl", "rust"]}>
```

Comment:
I removed rust from the unsupported list here:
https://github.com/getsentry/sentry-docs/pull/4610/files#diff-24efc89f65a56d014df9a3fad408e810af36da9e714249d342717f237242b53fR41
Must have been a merge conflict that is turned up again, can you make sure it is indeed supported?

Reply:
thanks for catching this, i noticed that the rust docs looked a little off but i wasn't sure if that was due to a merge conflict